tests/data/entities/test_data_access_layer.py

```diff
@@ -117,30 +117,6 @@ def test_dataset_data_write_output_file_should_write_file():
     assert path.is_file()
 
 
-def test_dataset_data_write_output_file_should_check_if_path_exists():
```

Comment:
yay clean up! :) 
do we need a test to check that the folder is indeed created now (also nested folders?)

Reply:
done by changing existing test :)